ePBFT/src/pbft/libbyz/LedgerReplay.h

```diff
@@ -24,9 +24,9 @@ class LedgerReplay
   std::unique_ptr<T> create_message(
     const uint8_t* message_data, size_t data_size);
   size_t cursor() const;
-  void apply_data(
+  std::unique_ptr<Pre_prepare> apply_data(
```

Comment:
Should the function name change? it seems odd that you apply data and then are given a Pre_Prepare message back

Reply:
Yes!